### ISSUE TITLE
ExtendedTask not allowed in modules

### ISSUE THREAD

**Issue by bernorev** [posit-dev/py-shiny#1115]
Hi,

I am having issues using an extended task within a module as a result of how module component id's are concatenated.

For example if there is a dashboard page which contains a module with a input_task_button (id = "files_submit"), the id will be 
```
dashboard_page-files_submit
```

which results in the error : 

```
_send_error_response: The string 'dashboard_page-files_submit' is not a valid id; only letters, numbers, and underscore are permitted
```

Is this a know issue with a workaround?

**Comment by gadenbuie** [comment]
Thanks for the report! We recently addressed this issue in #1108. The fix will be included in the next release of Shiny, and is currently available in the development version on GitHub.

**Comment by gadenbuie** [comment]
I'm pretty certain this is a duplicate of #1107, so I'm going to close the issue. Feel free to reply if they are different issues.
